python/sglang/srt/layers/quantization/mxfp4_utils.py

```diff
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: Apache-2.0
+import torch
+
+OCP_MX_BLOCK_SIZE = 32
+
+
+def dequant_mxfp4(x: torch.Tensor, scale: torch.Tensor, float_dtype: torch.dtype) -> torch.Tensor:
+    try:
+        from quark.torch.kernel import mx
+    except ImportError as err:
+        raise ImportError("The package `amd-quark` is required to use "
+                        "MX-FP4 models. Please install it with `pip install "
+                        "amd-quark`.") from err
```

Comment:
Pls modify `python/pyproject.toml` to have `amd-quark` (with a versioned one) installed to hip case.